doc/source/parameters/cfd/nitsche.rst

```diff
@@ -51,7 +58,7 @@ These parameters are used for simulations using the Nitsche immersed boundary me
       # Force and torque calculation on solid
       set calculate force on solid = false
       set solid force name = force_solid
-      set calculate torque on solid = true
+      set calculate torque on solid = false
```

Comment:
I guess you can remove all the parameters that are false by default!

Reply:
This is in the documentation, not in the example :) I saw this error while I was checking the parameters of the example (I removed the parameters that are used at their default values).